src/bin/main.rs

```diff
@@ -142,7 +147,8 @@ fn run_chickadee(ips: String, columns: Option<Vec<String>>) -> Result<()> {
     Ok(())
 }
 
-#[cfg(test)]
+/*
+[cfg(test)]
```

Comment:
We will need to re-enable the tests ;) 

We should also add tests for the new CLI arg and `resolver` parameter.